pkg/processor/build/builder.go

```diff
@@ -1342,6 +1352,25 @@ func (b *Builder) getProcessorDockerfileOnbuildImage(runtimeDefaultOnbuildImage
 	return runtimeDefaultOnbuildImage, nil
 }
 
+func (b *Builder) getDockerFileBuildArgs() map[string]string {
+
+	// Get platform build args for our runtime
+	buildArgs := b.platform.GetBuildArgs(b.runtime)
```

Comment:
when creating the `b.options` which is a  `CreateFunctionBuildOptions` instance, you can simple pass the runtime configuration along, and get on it, e.g.:

```
b.runtime.GetBuildArgs(b.options.GetRuntimeConfiguration)
```

that way, you dont need platform to have a function that serve _builder_ only. you simply delegate the configuration and let _builder_ resolve it with _runtime_ by itself. wtyt?

Reply:
The default build args are in platform configuration which are not part of `CreateFunctionBuildOptions`
Function build options are just a failback, 
so I don't see how will this work